### ISSUE TITLE
[Bug] BetterMic plugin causes Discord not to connect to voice channel. 

### ISSUE THREAD

**Issue by jz5678910** [verticalsync/Sunroof#15]
### Discord Account

Jz5678910#9608

### Operating System

Windows 11 & SteamOS & Nobara 40

### Linux Only ~ Desktop Environment

Gnome on Wayland, Whichever KDE steam deck uses. 

### What happens when the bug or crash occurs?

When the better Mic/Phil's Mic is active, Suncord gets stuck on connecting to RTC channel when trying to join a Voice channel. 

This happens across all three devices that I use Suncord on. 

Please correct me if this is not the right place, but the plugin is not found on any other client I've tried so I believe it's related to Suncord/Sunroof. 

### What is the expected behaviour?

I expect to be able to join a Voice channel correctly. 

### How do you recreate this bug or crash?

1. Enable better mic plugin
2. Restart Suncord
3. Join a Voice channel

### Debug Logs

Will try to update when I have access to my device. 

### Request Agreement

- [X] I have searched the existing issues and found no similar issue
- [X] I am using the latest Sunroof and Suncord versions
- [X] This issue occurs on an official release (not just the AUR packages)

**Comment by verticalsync** [comment]
yeah i think the plugin library is broken, so that may be why, ill check it out a bit later and let you know if i fixdd it.
